src/pages/control-user/ControlUser.tsx

```diff
@@ -17,14 +17,15 @@ import {
 } from '../../service/user/UserService';
 
 import { EditOutlined, KeyOutlined, LikeOutlined, DislikeOutlined } from '@ant-design/icons';
-
+import { useSearchParams } from 'react-router-dom';
 import ModalCreate from '../modal-create/ModalCreate';
 import ModalEdit from '../modal-edit/ModalEdit';
 import ModalReset from '../modal-reset/ModalReset';
+import ModalConfirm from '../modal-confirm/ModalConfirm';
 
 const statusStaffContent = [
```

Comment:
sao lại hardcode ở đây, cái này gọi API lấy về cơ mà

Reply:
ví dụ mà user hắn toàn là Active không, thì mình đâu có lấy đc giá trị Inactive để mình show ra đâu a

Reply:
done